cornac/datasets/README.md

```diff
@@ -0,0 +1,154 @@
+# Datasets
+
+For convenience Cornac offers easy access to a number of popular benchmark datasets. These are listed below along with their basic characteristics, followed by a usage example. In addition to preference feedback, some of these datasets come with item and/or user auxiliary information, which are grouped into three main categories:
```

Comment:
For easy experimentation, Cornac offers access to a number of popular recommendation benchmark datasets. These are listed below along with their basic characteristics, followed by a usage example. In addition to preference feedback, some of these datasets come with the item's and/or user's auxiliary information, which are grouped into three main categories:

Reply:
Thanks, I revised the beginning. I choose not to use `'s` in the last sentence, it is not required here.